spec/models/organisation_spec.rb

```diff
@@ -291,6 +291,15 @@ def build_org_with_computed_fields_and_updated_at org, updated_at = nil
       }).to raise_error CSV::MalformedCSVError
     end
 
+    it 'should be able to substitute with empty string when data is missing' do
+      attributes = {name: '',
+                    address: '',
+                    description: '',
+                    postcode: '',
+                    website: '',
+                    telephone: ''}
+      expect { Organisation.create_and_substitute_with_empty attributes }.not_to raise_error
```

Comment:
we could say described_class here ... my miss ... although I have mixed feelings about it's use anyway :-)